src/datatypes/src/vectors.rs

```diff
@@ -174,8 +174,19 @@ pub trait MutableVector: Send + Sync {
 
     /// Push value ref to this mutable vector.
     ///
-    /// Returns error if data types mismatch.
-    fn push_value_ref(&mut self, value: ValueRef) -> Result<()>;
+    /// # Panics
+    /// Panics if error if data types mismatch.
```

Comment:
Aren't these `push_value_ref`'s docs?

Reply:
Based on the above, I should update the docs and move this from `try_push_value_ref` to `push_value_ref`.

Reply:
Please move the `# Panics` section to `push_value_ref`.

Reply:
sorry, my bad, should be fixed now